runtime/src/main/java/org/corfudb/runtime/collections/streaming/StreamingManager.java

```diff
@@ -15,17 +15,12 @@
 import org.corfudb.util.Utils;
 
 import javax.annotation.Nonnull;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.UUID;
+import java.util.*;
```

Comment:
here as well.  There must a setting in the IDE to avoid wildcard imports

Reply:
not sure what is the setting yet. need to google it out. For now, I have removed them manually.